doc/rfcs/004-quickly-collaborate-with-coworkers-and-friends.md

```diff
@@ -0,0 +1,168 @@
+6. **Add support for audio before adding in-Atom portal invitations**: Teletype could add support for audio before adding support for establishing collaboration sessions directly within Atom. When collaborating with people in different physical locations, you'll likely need audio support in order to collaborate. Since Teletype doesn't currently provide audio support, these collaborators will still need a third-party solution for audio. If you're already relying on a third-party solution for audio, you can often use that same solution to send your collaborators your portal URL, which reduces the need for Teletype to maintain a list of trusted collaborators. However, there are still many [scenarios where it's helpful to quickly invite a trusted collaborator even if Teletype doesn't yet support audio](https://github.com/atom/teletype/pull/344#discussion_r175319913).
+
+##### What is the impact of not doing this?
+
+People will collaborate less often. Given the additional steps needed to start collaborating, there will be more instances where people decide that it's not worth the effort (i.e., people will enjoy rich collaboration less often).
+
+## Unresolved questions
+
+##### What unresolved questions do you expect to resolve through the RFC process before this gets merged?
+
+- If I invite a trusted collaborator to join my portal, and that person has multiple Atom windows open, can we show the invitation only in the frontmost window?
+- If I invite a trusted collaborator to join my portal, and that person has Atom windows open on multiple computers, are we OK with showing the invitation on each computer?
+- Where will the lists of recent and trusted collaborators appear in the UI? How do I show/hide these lists?
+
+##### What unresolved questions do you expect to resolve through the implementation of this feature before it is released in a new version of the package?
+
+- What architecture/services/libraries will we use to communicate portal invitations?
+- In order to provide the functionality described above, does teletype-server need to persist your list of trusted collaborators, or can we meet these needs while only storing this data locally?
+- If the host invites a trusted collaborator to their portal and then closes the portal before the invitee joins the portal, what should an invitee see/experience to inform them that the portal no longer exists?
+
+##### What related issues do you consider out of scope for this RFC that could be addressed in the future independently of the solution that comes out of this RFC?
+
+See [out of scope](#out-of-scope) section above.
+
+---
+
+[A] A user is **online** if they have Atom open, and Teletype is installed, and they are signed into Teletype, and their network allows them to successfully communicate with `api.teletype.atom.io`. In all other situations, the user is considered to be **offline**.
```

Comment:
> their network allows them to successfully communicate with `api.teletype.atom.io`.

What about if the computer goes to sleep, will a user be shown as offline?

Reply:
Yes. If your computer goes to sleep, we'll want Teletype to show you as offline.